src/styleguide/_patterns/02-molecules/02-locations/location-card.mustache

```diff
@@ -1,17 +1,11 @@
 <div class="grav-c-location-card">
-  <div class="grav-c-location-card__img">
-    <img src="{{location.img}}" alt="{{location.img-alt}}">
-  </div>
-  <div class="grav-c-location-card__content">
-    <h2>{{location.name}}</h2>
-    <p>{{#location.addressline1}}{{location.addressline1}}<br>{{/location.addressline1}}
-      {{#location.addressline2}}{{location.addressline2}}<br>{{/location.addressline2}}
-      {{#location.city}}{{location.city}}<br>{{/location.city}}
-      {{#location.postcode}}{{location.postcode}}<br>{{/location.postcode}}
-      {{#location.country}}{{location.country}}{{/location.country}}
-    </p>
-  </div>
-  <div class="grav-c-location-card__cta">
-    {{> atoms-link:grav-c-block-link }}
-  </div>
+  <h2>{{location.name}}</h2>
+  <address>
+    {{#location.addressline1}}{{location.addressline1}}<br>{{/location.addressline1}}
+    {{#location.addressline2}}{{location.addressline2}}<br>{{/location.addressline2}}
+    {{#location.city}}{{location.city}}<br>{{/location.city}}
+    {{#location.postcode}}{{location.postcode}}<br>{{/location.postcode}}
+    {{#location.country}}{{location.country}}{{/location.country}}
+  </address>
+  <img src="{{location.img}}" alt="{{location.img-alt}}">
```

Comment:
The image should surely be inside an anchor and point to google maps? I'm a bit unsure about using a screen grab of google maps anyway, It's misleading and looks like it should behave like Google maps. Especially if you using touch gestures.

Reply:
...having said that, it's exactly what Mozilla do! e.g. https://www.mozilla.org/en-US/contact/spaces/mountain-view/

Reply:
Is not a screen grab from Google Maps. Is an API made to show a map but to avoid loading the JS bundle if not necessary. The images can be cached and they do the job, especially in a page like this where we want to show many different ones.

The anchor tag is ready now, I wanted to receive feedback on the design asap.

Reply:
👍